### DISCUSSION TITLE
What's our code review process like?

### DISCUSSION THREAD

**Discussion by tpat95** [SCIInstitute/SCIRun#2191] [Q&A] [answered]
I'd like to contribute a bug fix. What can I expect?

**Comment by nids2001** [comment, marked as answer]
SCIRun uses the [Github flow model](https://guides.github.com/introduction/flow/) with at least two reviews per pull request, as well as a suite of checks run by Travis and Github Actions. Typically the code is reviewed for style and correctness within a few days, but if there are multiple open PRs it might take longer
